sklearn/metrics/cluster/_expected_mutual_info_fast.pyx

```diff
@@ -35,36 +35,32 @@ def expected_mutual_information(contingency, int n_samples):
     nijs = np.arange(0, max(np.max(a), np.max(b)) + 1, dtype='float')
     nijs[0] = 1  # Stops divide by zero warnings. As its not used, no issue.
     # term1 is nij / N
-    term1 = nijs / N
+    term1 = nijs.base / n_samples
```

Comment:
Based on https://github.com/scikit-learn/scikit-learn/pull/25713/files#r1118740841, should this be `nijs.base` or just `nijs` (because `np.arange()` doesn't return a memoryview?

Reply:
here `nijs` is a memoryview. This is the cython way of taking memoryview from ndarrays
```
cnp.float64_t[::1] nijs
nijs = <some ndarray>
```

Reply:
see https://cython.readthedocs.io/en/latest/src/userguide/memoryviews.html#using-memoryviews